net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -16,7 +16,7 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ~ppc ~ppc64 x86 ~amd64-linux ~x86-linux"
 # IUSE="+cli" doesn't work due to https://bugs.gentoo.org/831045#c3
-IUSE="+asm +berkdb +bitcoin-cli +daemon dbus examples +external-signer kde libs +man nat-pmp +qrcode gui +sqlite system-leveldb +system-libsecp256k1 systemtap test upnp zeromq"
+IUSE="+asm +berkdb +bitcoin-cli +daemon dbus examples +external-signer gui kde libs +man nat-pmp +qrcode +sqlite system-leveldb +system-libsecp256k1 systemtap test upnp zeromq"
```

Comment:
I know I am late to the party, but I just noticed that we probably could have named the `bitcoin-cli` USE flag simply `cli` (which is already a global USE flag). Or is there a reason to name it `cli`? Not saying that we have to change this in this PR. But this is maybe something to consider for later updates.

Reply:
Did you read the comment that is literally on the preceding line? 😉

Reply:
Ha, funny. Yes, I read it when doing the initial PR review, but then forgot about it.